Tools/jit/_llvm.py

```diff
@@ -21,9 +21,7 @@ def _async_cache(f: _C[_P, _R]) -> _C[_P, _R]:
     lock = asyncio.Lock()
 
     @functools.wraps(f)
-    async def wrapper(
-        *args: _P.args, **kwargs: _P.kwargs  # pylint: disable = no-member
```

Comment:
did you mean to get rid of various pylint suppression comments in this PR? I assume it was there because it was useful to the JIT developers working on this code

Reply:
Oops, I'd meant to add them back after reformatting. 

cc @brandtbucher to check that the comments are still useful (we don't run pylint in CI, currently)

Reply:
Can you add them back? I can remove them locally if needed, but I haven't updated Pylint recently or anything.

Reply:
Restored the comments and set back to 88 line length.

A